launchUI.py

```diff
@@ -2143,35 +2143,57 @@ def create_layout_pannel(master, x, width, height):
 
 def clickDisplayLayout():
     # Hardcodes the related path, will fix later.
-    test_module_name = "TileRTL__f6add65c4dde1319"
+    standard_module_name = "CGRATemplateRTL"

```

Comment:
Would this work with random ending, say, `CGRATemplateRTL__f6add65c4dde1319`?

Reply:
Yes, the top module name will be replaced with `CGRATemplateRTL` when generating the verilog file for OpenRoad. The corresponding code is `line = "module " + standard_module_name + " ("`

Reply:
Do we need to modify the [VectorCGRA](https://github.com/tancheng/VectorCGRA/blob/master/cgra/translate/CGRATemplateRTL_test.py) to make sure the generated verilog is named as `CGRATemplateRTL.sv` instead of `CGRATemplateRTL__f6add65c4dde1319.sv`? Or you rename the file name with some bash command in this script?

Reply:
Yeah it's better to modify the VectorCGRA because now the generated RTL is named as `design.v`. And I need to rename it to `CGRATemplateRTL.sv` first, then convert it to `CGRATemplateRTL.v`, finally modify the top module name. 

Don't worry, I will modify it these days.

Reply:
Thanks a lot!